packages/syntax-core/src/ThemeProvider/ThemeProvider.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useMemo } from "react";
+import variables from "@cambly/syntax-design-tokens/dist/json/variables.json";
+
+type ThemeName = "classic" | "cambio";
+
+type Theme = {
+  themeName: ThemeName;
+};
+
+const ThemeContext = React.createContext<Theme>({
+  themeName: "classic",
+});
+ThemeContext.displayName = "ThemeContext";
+
+const classicToCambioKeyLookup = {
+  "color-base-black": "color-cambio-black",
+  "color-base-destructive-100": "color-cambio-destructive-100",
+  "color-base-destructive-200": undefined, // Deprecated - to be deleted
```

Comment:
@ajmooo fyi still seeing quite a lot of usage of deprecated colors so just falling back to the legacy "classic" colors and will remove in a follow up